Sources/MapboxCoreNavigation/CoreFeedbackEvent.swift

```diff
@@ -2,10 +2,10 @@ import Foundation
 import MapboxDirections
 import Polyline
 
-class CoreFeedbackEvent: Hashable {
-    let id = UUID()
-    let timestamp: Date
-
+class CoreFeedbackEvent: Hashable, Codable {
+    var identifier = UUID()
```

Comment:
What mutability gives us here?

Reply:
Why the rename from id to identifier?

Reply:
I renamed the field in accordance to our Swift style guide.
Also removed mulability.